padinfo/view_state/transforminfo.py

```diff
@@ -0,0 +1,34 @@
+from padinfo.common.config import UserConfig
+from padinfo.core.transforminfo import perform_transforminfo_query
+from padinfo.view_state.base import ViewState
+
+
+class TransformInfoViewState(ViewState):
+    def __init__(self, original_author_id, menu_type, raw_query, color, b_mon, t_mon,
+        base_rarity, acquire_raw, true_evo_type_raw):
+        super().__init__(original_author_id, menu_type, raw_query, extra_state=None)
+        self.color = color
+        self.b_mon = b_mon
+        self.t_mon = t_mon
+        self.base_rarity = base_rarity
+        self.acquire_raw = acquire_raw
+        self.true_evo_type_raw = true_evo_type_raw
+
+    def serialize(self):
+        ret = super().serialize()
+        ret.update({
+            'b_query': str(self.b_mon.monster_id),
```

Comment:
and these are `b_resolved_monster_id` not `query` right?

Reply:
hmmm. I copied the way ls did it here, partly just to imitate style and partly because the check in id's deserialize only looks for `query` to determine what to show. using the monster id as query will surely get the right monster back. I'd worry it might get clunky in the future if another command shows up with `[another letter]_resolved_monster_id`

mmmm actually thinking about it more it gets renamed to `query` in tf menu so the interim name can be something else for clarity. will change